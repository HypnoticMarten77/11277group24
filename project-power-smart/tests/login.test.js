import 'react-native';
import React from 'react';
import loginPage from '../components/LoginPage'
import renderer from 'react-test-renderer'
import {Image, Text} from 'react-native'
import styles from "../StyleSheets/AppStyling";
jest.mock('react-native-vector-icons/SimpleLineIcons', () => 'Icon')
jest.mock('react-native-vector-icons/FontAwesome5', () => 'Icon')
it('renders correctly', () => {
    const tree = renderer.create(
        <loginPage/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
it('image is correct', () => {
    const tree = renderer.create(
        <Image style={styles.image} source={require("../assets/powersmart.png")}/>
        ).toJSON();
    expect(tree).toMatchSnapshot();
})