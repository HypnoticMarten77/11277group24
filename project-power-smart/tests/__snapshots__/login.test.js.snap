// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `<loginPage />`;

exports[`renders text 1`] = `
<Image
  source={
    Object {
      "testUri": "../../../assets/powersmart.png",
    }
  }
  style={
    Object {
      "height": 200,
      "marginTop": -50,
      "resizeMode": "contain",
      "width": 200,
    }
  }
/>
`;
